docs/install.rst

```diff
@@ -131,21 +130,34 @@ Finally, you're ready to start the webserver::
 
     python manage.py runserver
 
-Visit http://127.0.0.1:8000/ in your browser to see how it looks; you can use
-the admin interface via http://127.0.0.1:8000/admin (logging in with the
-superuser account you just created).
+Visit http://127.0.0.1:8000/ in your browser to see how it looks;
+you can use the admin interface via http://127.0.0.1:8000/admin
+(logging in with the superuser account you just created).
 
-For builds to properly kick off as expected, it is necessary the port
-you're serving on (i.e. ``runserver 0.0.0.0:8080``) match the port defined
-in ``PRODUCTION_DOMAIN``. You can utilize ``local_settings.py`` to modify this.
+For builds to properly work as expected,
+it is necessary the port you're serving on
+(i.e. ``python manage.py runserver 0.0.0.0:8080``)
+match the port defined in ``PRODUCTION_DOMAIN``.
+You can utilize ``readthedocs/settings/local_settings.py`` to modify this.
 (By default, it's ``localhost:8000``)
 
-While the webserver is running, you can build documentation for the latest version of
-a project called 'pip' with the ``update_repos`` command.  You can replace 'pip'
-with the name of any added project::
+While the webserver is running,
+you can build the documentation for the latest version of any project using the ``update_repos`` command.
+for example to update the ``pip`` repo::
 
     python manage.py update_repos pip
 
+Further steps
```

Comment:
With this steps you can start the server and do a build (sometimes is required to install some more dependencies for generating the pdf), but I feel that these others docs are connected on some way

Reply:
I'd say that this section should go _after_ "What's available".

Reply:
Also, in the section "What's available" you could include a note mentioning that `pdf` generation is not covered here or to do that it's required to use the docker image.